extensions/stackdriver/log/logger_test.cc

```diff
@@ -161,7 +161,7 @@ std::string write_audit_request_json = R"({
            "referer":"www.google.com",
            "serverIp":"2.2.2.2",
            "latency":"10s",
-           "protocol":"HTTP"
+           "protocol":"http"
```

Comment:
are we concerned here about the possibility of breaking any filters, etc. ?

Reply:
I think our actual extensions already send lowercase strings based on integration tests. I just made the unit test match it.